src/App.js

```diff
@@ -36,6 +38,7 @@ class App extends React.Component {
           <Header user={this.state.user} onLogout={this.logoutHandler} />
           <Switch>
             <Route exact path="/">
+              <BestBooks></BestBooks>
```

Comment:
Good work on the lab so far! You mentioned you had some questions on conditional rendering:

- In state we have the `user`
- we can use ternary to do this in React  [conditional rendering React](https://reactjs.org/docs/conditional-rendering.html)
- If `this.state.user ?` then show the BestBooks component else the Login component

See what you can get workin' 👍